Automattic/jetpack
# My Jetpack:  add plan section component to storybook

### Depends on
* ~https://github.com/Automattic/jetpack/pull/22319~
* ~https://github.com/Automattic/jetpack/pull/22302~

This PR adds the `<PlanSection />` component to the storybook.

![image](https://user-images.githubusercontent.com/77539/149201433-760a48ab-5e38-4758-862a-b267f9329de6.png)

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack:  add plan section component to storybook

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Startup the storebook

```cli
cd projects/js-packages/storybook
```

```cli
pnpm install & pnpm storybook:dev
```

* confirm you see the `Plan Section` item in the sidebar.
* confirm you see the component rendered.

_Note: the data becomes from the mocked `/my-jetpack/site` endpoint_
